.github/workflows/build.yaml

```diff
@@ -21,6 +22,13 @@ jobs:
       - name: Checkout Code
         uses: actions/checkout@v2
 
+      - name: SonarQube Scan
+        uses: docker://sonarsource/sonar-scanner-cli:latest
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
What does the scanner use the `GITHUB_TOKEN` for?

Reply:
This was copy/pasted from the Sonar docs.  I assumed it was for pull request decoration that I didn't set up yet, but upon further inspection, it doesn't look like their Docker image uses this env variable at all.  I'll remove it.